apps/web/components/v2/apps/CalendarListContainer.tsx

```diff
@@ -141,6 +141,7 @@ function CalendarList(props: Props) {
   );
 }
 
+// todo: @hariom extract this into packages/apps-store as "GeneralAppSettings"
```

Comment:
hey @hariombalhara this would be awesome to be abstracted into something like "GeneralAppSettings"